test/suites/proxy.sh

```diff
@@ -0,0 +1,78 @@
+test_proxy_device() {
+  
+  MESSAGE="Proxy device test string"
+
+  lxc launch busybox proxyTester
+  lxc config device add proxyTester proxyDev proxy listen=tcp:127.0.0.1:1234 connect=tcp:127.0.0.1:4321 bind=host
+  if [ $(lxc config device list proxyTester) != "proxyDev" ]; then
+    echo "Proxy device was not added to container"
```

Comment:
You should then call "false" afterwards to cause the error to actually be fatal, otherwise the message will be printed but the test won't fail.